hubblestack/extmods/modules/pulsar.py

```diff
@@ -395,3 +395,44 @@ def _dict_update(dest, upd, recursive_update=True, merge_lists=False):
             for k in upd:
                 dest[k] = upd[k]
         return dest
+
+#TODO:change paths for top.pulsar and config yaml to 'salt://hubblestack_pulsar/top.file'
+
+def top(topfile='/root/myhubblefiles/topfile/top.pulsar',
```

Comment:
Just reiterating your TODO. The default path should probably be `salt://hubblestack_pulsar/top.pulsar`. On the windows side, it will be `salt://hubblestack_pulsar/win_top.pulsar`.